source/StellarObject.h

```diff
@@ -72,10 +74,17 @@ class StellarObject : public Body {
 	// Find out how far this object is from its parent.
 	double Distance() const;
 
+	// Change the distances an object is visible from, based on flagship attributes.
```

Comment:
What are these flagship attributes?

Reply:
Currently, nothing. It's present for future functionality that doesn't have a well-defined syntax yet, because the exact needs aren't completely clear.

Reply:
As I suggested in another comment: Maybe better to leave those functions out completely, since they are currently not used.